app/apptesting/test_suite.go

```diff
@@ -47,6 +51,15 @@ func (s *KeeperTestHelper) Setup() {
 	s.TestAccs = CreateRandomAccounts(3)
 }
 
+func (s *KeeperTestHelper) CreateTestContext() sdk.Context {
```

Comment:
I don't think this is used anywhere. Let's remove it?

Reply:
Added it since I thought we would use it one day :p Agreed to removing it for now, But we should definitely utilize the apptesting package better!